src/main/java/com/capitalone/dashboard/collector/DefaultGitHubClient.java

```diff
@@ -377,7 +377,7 @@ public RedirectedStatus checkForRedirectedRepo(GitHubRepo repo) throws Malformed
         JSONObject queryJSONBody = parseAsObject(response);
         String repoUrl = str(queryJSONBody, "html_url");
         if (!repoUrl.equals(repo.getRepoUrl())) {
-            LOG.info("original url: " + repo.getRepoUrl() + " is redirected to new url: " + repoUrl);
+            LOG.info("original_url=" + repo.getRepoUrl() + ", redirected_url: " + repoUrl);
```

Comment:
do you want to change the : to = for easy Splunk search?

Reply:
sure